src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  const newObj = Object.assign({}, state);
```

Comment:
```suggestion
  const newObj = Object.assign({}, state);
```

Use better naming,  for example stateClone or stateCopy